docs/iohk-nix.md

```diff
@@ -74,7 +74,7 @@ let
     # The overlay allows extension or restriction of the set of
     # packages we are interested in. By using the stack-pkgs.overlay
```

Comment:
Shouldn't the lines above not include "overlay" as well?

Reply:
This one should go.